internal/app/frontend/frontend_service_test.go

```diff
@@ -79,6 +81,87 @@ func TestDoCreateTickets(t *testing.T) {
 	}
 }
 
+func TestDoGetAssignments(t *testing.T) {
+	testTicket := &pb.Ticket{
+		Id: "test-id",
+	}
+
+	totalAssignments := 2
+	var testAssignments []*pb.Assignment
+
+	for i := 0; i < totalAssignments; i++ {
+		testAssignments = append(testAssignments, &pb.Assignment{Connection: string(i)})
+	}
+
+	tests := []struct {
+		description      string
+		count            int
+		preAction        func(*testing.T, statestore.Service)
+		senderGenerator  func(chan *pb.Assignment, *int) func(*pb.Assignment) error
+		shouldCode       codes.Code
```

Comment:
This is a little confusing. Here is a suggestion to make it more readable: use got / want variables (gotCode, wantCode .. gotAssignment, wantAssignment..) to track stuff you expect as result (in want) and you actually get (in got variables) 

In this case, the testAssignments should be wantAssignment and set in each case. shouldCode is wantCode, shouldCount may not be necessary (can just be len of wantAssignment). shouldAssignment is then gotAssignment only stuff you are getting back - can initialize in the calling test case - no need to be a part of the test state.

Reply:
You are right, count is not needed. Comments resolved.